packages/assets-controllers/src/NftDetectionController.test.ts

```diff
@@ -1588,7 +1580,10 @@ describe('NftDetectionController', () => {
             `/collections?contract=0xebE4e5E773AFD2bAc25De0cFafa084CFb3cBf1eD&chainId=1`,
           )
           .replyWithError(new Error('Failed to fetch'));
-        await Promise.all([controller.detectNfts(), controller.detectNfts()]);
+        await Promise.all([
+          controller.detectNfts(['0x1']),
```

Comment:
nit - now that we can detect with multiple chains, can we have a test when entering multiple chain params?

Reply:
Fixed here [912bed2](https://github.com/MetaMask/core/pull/5448/commits/912bed258bf30ef33802350454e95799ca9433e7)